ruleset/rules/0840-win_event_channel.xml

```diff
@@ -1,79 +1,84 @@
-    <rule id="92036" level="6">
-        <if_sid>92035</if_sid>
-        <field name="win.eventdata.authenticationPackageName" type="pcre2">NTLM</field>
-        <description>Successful Remote Logon Detected - NTLM authentication, possible pass-the-hash attack</description>
-        <mitre>
-            <id>T1550.002</id>
-        </mitre>
-        <group>authentication_success,pci_dss_10.2.5,gpg13_7.1,gpg13_7.2,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
-    </rule>
+  <rule id="92653" level="0">
```

Comment:
According to the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html), a successful login should be level 3.
```suggestion
  <rule id="92653" level="3">
```